SendGrid/Example/Example.cs

```diff
@@ -35,13 +35,13 @@ private static void Main()
             String subject = "I'm replacing the subject tag";
             Email to = new Email("elmer@sendgrid.com");
             Content content = new Content("text/html", "I'm replacing the <strong>body tag</strong>");
-            Mail mail = new Mail(from, subject, to, content);
+            SendGrid.Models.Mail mail = new SendGrid.Models.Mail(from, subject, to, content);

```

Comment:
I'm thinking we move this code behind the scenes.

I think something like this would replace this example:

```csharp
String apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY", EnvironmentVariableTarget.User);
email = new Email(SendGrid.SendGridAPIClient(apiKey));
// The internal Email object would be renamed to Contact and the object would be instantiated behind the scenes, same with all the other getters and setters.
email.SetFrom("dx@sendgrid.com");
email.AddSubject("I'm replacing the subject tag");
email.AddContent("text/html", "I'm replacing the <strong>body tag</strong>");
email.SetTemplateID("13b8f94f-bcae-4ec6-b752-70d6cb59f932");
email.AddSubstitution("-name-", "Example User");
email.AddSubstitution("-city-", "Denver");
if(email.Verify() == True){
  response = email.Send();
}
Console.WriteLine(response);
Console.ReadLine();
```